src/test/FuzzerImpl.cpp

```diff
@@ -1027,13 +1027,29 @@ std::array<ClaimableBalanceParameters, 3> constexpr claimableBalanceParameters{{
     {4, 5, AssetID(4), 20, 6} // sponsored by account 6
 }};
 
-struct OfferParameters
+struct OfferParameters : public SponsoredEntryParameters
 {
     constexpr OfferParameters(int publicKey, AssetID const bid,
```

Comment:
Nit: `bid` and `sell` can be `const&` here. Same with the constructor below.

Reply:
Changed.